conf/modules/call_repeat_expansions.config

```diff
@@ -58,11 +58,10 @@ process {
         ]
     }
 
-    withName: '.*:CALL_REPEAT_EXPANSIONS:BCFTOOLS_MERGE' {
+    withName: '.*:CALL_REPEAT_EXPANSIONS:TRGT_MERGE' {
         ext.prefix = { "${meta.id}_repeats" }
         ext.args = [
             '--output-type z',
-            '--write-index=tbi',
```

Comment:
Since this no longer writes an index, we need to index the file and publish when annotate repeat expansions is not active.